configure.ac

```diff
@@ -1516,6 +1521,16 @@ else
   with_python=""
 fi
 
+if test "x$enable_python" = "xyes"; then
+    if test "x$with_python_packages" = "xvenv"; then
+        PYTHON_VENV_DIR=`pwd`/venv
+        PYTHON_VENV=${PYTHON_VENV_DIR}/bin/python
+    else
+        with_python_packages="system"
+        PYTHON_VENV=$PYTHON
```

Comment:
We don't have to use $PYTHON_VENV everywhere

$PYTHON => system python
$PYTHON_VENV => Python in our build-time virtualenv

The func_test does not have any external dependencies, so it does not have to run in our virtualenv, it can use the system one.

We are installing PyPI packages into our venv but not to the system.

Does that make sense? I was trying to use a name that encodes that this is different from system Python.

To complicate stuff further, we will have another virtualenv for production purposes, which is different from our build-time one.

Reply:
Yes, it makes sense, thanks! :)